# set loop rate and if cv image should be showed
loop_rate: 10
show_image: true

#Camera topic here
camera_topic: "/tello/camera/image_raw"
queue_size: 1

# Set your detection topic here
detection_topic: "/darknet_ros/bounding_boxes"

# Set your output topic here
tracker_topic: "/sort_tracker/tracked_bounding_boxes"

# Threshold for Costs between detections and tracks
cost_threhold: 15

# min_hits
min_hits: 1

# max age
max_age: 5

image_view:

  enable_opencv: true
  wait_key_delay: 1
  enable_console_output: true
